src/edu/csus/ecs/pc2/core/imports/clics/EventFeedLoader.java

```diff
@@ -0,0 +1,50 @@
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * 
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class EventFeedLoader {
+
+    private static ObjectMapper mapper = new ObjectMapper();
+    
+    static {
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    }
+
+    public static List<TeamAccount> createTeamAccounts (String [] lines) throws JsonParseException, JsonMappingException, IOException {
```

Comment:
I suggest this method be renamed to "getTeamAccounts".  I think "createTeamAccounts" makes it easy to misinterpret that this method will somehow actually create accounts in PC2.

Reply:
renamed.